x-pack/legacy/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -101,6 +131,13 @@ export function dataFrameAnalyticsRoutes({ xpackMainPlugin, router }: RouteIniti
     })
   );
 
+  /**
+   * @api {put} /api/ml/data_frame/analytics/:analyticsId Gets stats for requested analytics job
```

Comment:
Copy and paste error? Isn't this endpoint used for creating a new analytics job?